tests/test_codec.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+
+from multiaddr.codec import size_for_addr
+from multiaddr.codec import bytes_split
+from multiaddr.codec import string_to_bytes
+from multiaddr.codec import bytes_to_string
+from multiaddr.codec import address_bytes_to_string
+from multiaddr.codec import address_string_to_bytes
```

Comment:
another nit which flake8 will point out: imports should be sorted alphabetically